docs/integrations/digitalocean/k8s.mdx

```diff
@@ -0,0 +1,281 @@
+
+   :::tip
+
+   Make sure you edit line `10` of the YAML below, which contains the `NGROK_DOMAIN` variable, with the ngrok domain you created earlier.
+
+   :::
+
+   ```yaml showLineNumbers
+   ---
+   apiVersion: networking.k8s.io/v1
+   kind: Ingress
+   metadata:
+     name: bookinfo-ingress
+     namespace: bookinfo
+   spec:
+     ingressClassName: ngrok
+     rules:
+       # highlight-start
+       - host: NGROK_DOMAIN
+         # highlight-end
+         http:
+           paths:
+             - path: /
+               pathType: Prefix
+               backend:
+                 service:
+                   name: productpage
+                   port:
+                     number: 9080
+   ```
+
+1. Apply the new configuration to your cluster:
+
+   ```bash
+   kubectl apply -f ngrok_ingress.yaml
+   ```
+
+   :::tip
+
+   **Note:** If you get an error when applying the manifest, double check that you've updated the `NGROK_DOMAIN` value and try again.
+
+   :::
+
+1. Navigate to `https://NGROK_DOMAIN` in your browser to see your example Bookinfo app as managed, deployed, and publicly networked via DigitalOcean, Kubernetes, and ngrok! Click **Normal user** to see a little more—it's not much to look at, but it's a working Kubernetes app with functional, secure, and simple ingress with the ngrok Kubernetes Operator.
+
+   ![Successfully deployed Booklist app on DigitalOcean, Kubernetes, and ngrok](img/bookinfo-deployed.png)
+
+## **Step 4**: Enable extra features of ngrok's Cloud Edge {#enable-extra-features-of-ngrok}
+
+To demonstrate how both [fanout ingress](/docs/k8s/with-edges.mdx#simple-fanout) and [OAuth support](/docs/http/oauth.mdx) work, you can quickly launch an open-source observability stack including [Prometheus](https://github.com/prometheus), [Grafana](https://github.com/grafana/grafana), and [Alertmanager](https://github.com/prometheus/alertmanager/) on your cluster.
+
+:::note
+
+Installing this monitoring stack is also possible using `doctl kubernetes 1-click install YOUR_CLUSTER --1-clicks netdata`, although at the time of writing, DigitalOcean's API did not respond as expected to complete the operation.
+
+:::
+
+1. Head over to the [Kubernetes Clusters](https://cloud.digitalocean.com/kubernetes/clusters/) dashboard on the DigitalOcean console and click on the cluster you created previously.
+
+1. Click the **Marketplace** tab, find or search for **Kubernetes Monitoring Stack**, and click **Install**. Confirm your choice.
+
+   In the background, DigitalOcean uses Helm to create a new integrated deployment for Prometheus, Grafana, and Alertmanager.
+
+1. Create a new domain in the [ngrok dashboard](https://dashboard.ngrok.com/cloud-edge/domains) at `monitoring.NGROK_DOMAIN`.
```

Comment:
Won't they also need to create the subdomain with their domain registrar? 🤔

Reply:
They wouldn't have to! I actually tested this and it worked fine. I first created an ngrok domain like `digitalocean.ngrok.app` for the initial deployment, then created another one at `monitoring.digitalocean.ngrok.app` for this observability deployment.

Is that clear the way I've described it? Maybe I could be clearer about what I did, or suggested creating a domain like `monitoring-digitalocean.ngrok.app` instead.

Reply:
🤔 Got it. With that context, that makes sense. I was looking at it through the "custom domain" lens. But, we never mention those in the scope of this article. So, I think we're good. Thanks for the explanation!